src/test_suite/fixture_utils.py

```diff
@@ -118,3 +128,30 @@ def extract_instr_context_from_fixture(fixture_file: Path):
         return 0
 
     return 1
+
+
+def get_program_type(instr_fixture: pb.InstrFixture) -> str:
```

Comment:
program type or name?

Reply:
It also classifies BPF programs so type is probably more fitting